torch/fx/experimental/proxy_tensor.py

```diff
@@ -93,6 +93,17 @@ def get_proxy(obj):
 def has_proxy(obj):
     return get_proxy(obj) is not None
 
+def set_meta(proxy, val):
```

Comment:
Do we also want an API to get meta for fx.Node? Otherwise user would need to look through three different fields.

Reply:
Non-blocking, but we probably should do it in latter PR.